### ISSUE TITLE
Use ASW for Amphibia

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#386]
The [ASW](https://amphibiansoftheworld.amnh.org) is pushing their data to ITIS in a very infrequent schedule. ITIS is behind several years already. It would be much better to have ASW as a direct source of COL, as it can be more up to date and have more original date incl links to the ASW site.

ASW is happy to publish their data as ColDP to CLB.

Amphibia in COL: https://www.catalogueoflife.org/data/taxon/PH

**Comment by DaveNicolson** [comment]
The amphibian data in ITIS was what ASW had as of September 2019, so I am not sure "behind several years already" is quite the case, but you're right that it is not current. Having worked with ASW data personally while preparing the 2019 update (and prior updates), I am not sure it will all come out smoothly as an automated update, but I guess you're welcome to try and see what happens. We did **a lot** to get the data to fit cleanly into the ITIS structure, and to deal with homonyms and other problematic names, and to ensure taxon authors for names in synonymy come out properly (they do not store the authorship that would apply to _non-original_ combinations (of the current name or any of its synonyms, and calculating it has significant pitfalls due to the complexity of some name set usages in synonymy). It will likely not be an easy task, I think.
Dave

**Comment by DaveNicolson** [comment]
To clarify, ASW doesn't "push" their data into ITIS, they make it available to us on demand. But as each Amphibia update in ITIS takes a significant amount of time to prepare, we don't do it on a running basis, but rather update it periodically. There was a long time between updates (major updates from ASW to ITIS were done in 2005, 2009, and 2019). We intend to update it more frequently in ITIS going forward, FWIW. -Dave

**Comment by mdoering** [comment]
Thanks Dave, that is good to know. I entirely leave the decision if COL adopts ASW directly to @yroskov. I was just being approached and generally like the idea of having access to original sources. Even if we can just compare COL/ITIS with the latest ASW in ChecklistBank should be useful.

**Comment by yroskov** [comment]
For attention of Taxonomy Group - @dhobern

**Comment by dhobern** [comment]
Thanks - I appreciate the information and Dave's perspective. Clearly a truly automated process would be fantastic, but the pitfalls may be major. @mdoering - how did the offer to generate a COLDP file come? Do they think they understand what it means and did they indicate whether they saw it as difficult? We could work with them not only to get a rough version into COLDP but to get as many as possible of the issues Dave references handled during generation.

I don't think there's value in discussing this in the Taxonomy Group until we can answer these aspects.

**Comment by yroskov** [comment]
Yes, data delivery in the checklistbank is an important issue (might be a barrier).

**Comment by mdoering** [comment]
Contact was made by ASW to GBIF, picking up very old ties. I showed the ColDP format to Matt Tarr (technical) and Darren Frost and they seemed confident to be able to generate the core set of data (taxa, references, english names & distribution) licensed with CC-BY. We agreed to start one by one, starting with just the taxonomic core and improving things iteratively once we have something in CLB. I will then show them the issue flagging. A first minimal version might be available in ~3 weeks.

**Comment by mdoering** [comment]
In the current situation ASW and Dr. Frost and nowhere mentioned in COL even though they are mostly responsible for all Amphibia content. Using them directly would therefore give them visibility and links to their site from COL.

I don't know how feasable this is for ITIS, but ColDP does allow to link each data record via the `sourceID` field to a detailed source citation in the [metadata.yaml](https://github.com/CatalogueOfLife/coldp/blob/master/metadata.yaml#L192). This would allow aggregated datasets to still cite their sources and not lose this information.

**Comment by DaveNicolson** [comment]
ASW is explicitly cited in ITIS for every single name ITIS got from them, along with a bunch of publications, but it looks like very few of the reference citations are coming through to COL. For example, compare:
_Gastrophryne carolinensis_ (Holbrook, 1835) ([TSN 173467](https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=173467#null))
vs:
https://preview.catalogueoflife.org/data/taxon/3FDVS

**Comment by yroskov** [comment]
@geoff, could you please review ITIS refs capture with Dave? 

Not sure, where we can accommodate ITIS data from _Other Sources_.
